app/src/main/java/com/asfoundation/wallet/home/ui/list/header/notifications/CardNotificationListModel.kt

```diff
@@ -1,20 +1,19 @@
-package com.asfoundation.wallet.home.transactions.models.cardnotifications
+package com.asfoundation.wallet.home.ui.list.header.notifications
 
 import com.airbnb.epoxy.EpoxyModel
 import com.airbnb.epoxy.EpoxyModelGroup
 import com.asf.wallet.R
-import com.asfoundation.wallet.home.transactions.models.DefaultCarouselModel_
+import com.asfoundation.wallet.home.ui.list.HomeListClick
 import com.asfoundation.wallet.referrals.CardNotification
-import com.asfoundation.wallet.ui.widget.holder.CardNotificationAction
 
 class CardNotificationListModel(val data: List<CardNotification>,
```

Comment:
the problem with the padding is in this file in the line 30
.numViewsToShowOnScreen(1.05f) // This should be consistent on every screen
if you remove it, the card_notification_list will be displayed in the center, i'm not sure if this is the best solution though, or if it has other implications...